discovery-infra/start_discovery.py

```diff
@@ -297,6 +298,13 @@ def _get_vips_ips(machine_net):
 # Converts params from args to assisted-service cluster params
 def _cluster_create_params():
     ipv4 = args.ipv4 and args.ipv4.lower() in MachineNetwork.YES_VALUES
+    ipv6 = args.ipv6 and args.ipv6.lower() in MachineNetwork.YES_VALUES
+    ntp_source = None
+    if ipv6 and not ipv4:
+        ntp_source = str(IPNetwork(args.vm_network_cidr6).ip + 1)
```

Comment:
Don't we already have the VM IP address calculated somewhere? Doing the same calculation in multiple places bothers me :/

Reply:
This is not VM IP, but the address on the host

Reply:
Doesn't matter, if we already calculate it somewhere else it should be re-used.

Reply:
Will it always be the IP on the host? Shouldn't we do something like `get_ip_by_interface_name`?

Reply:
@empovit Where else do we do this calculation?